### ISSUE TITLE
Example notebook reading parquet/csv files using duckdb

### ISSUE THREAD

**Issue by jcrist** [ibis-project/ibis-examples#3]
We should add an example notebook using duckdb to analyze data from a file source (parquet or csv). The duckdb backend dependencies are already installed in the binder image.

**Comment by ksuarez1423** [comment]
Do we need to focus on it being DuckDB? We do have the ibis.read_csv() route.

**Comment by jcrist** [comment]
If it uses `duckdb` I might mention it being duckdb behind the scenes. But no, the examples don't need to be "duckdb" examples per se, using the `ibis.read_*` apis would make sense here.

**Comment by cboettig** [comment]
Note that both of these fail:

```python
import ibis
con = ibis.duckdb.connect()
bucket = "gbif-open-data-us-east-1"
path = "occurrence/2023-02-01/occurrence.parquet/*"
con.register(f"s3://{bucket}/{path}")
```
error says "use read_parquet() etc" 
or 

```python
con.read_parquet(f"s3://{bucket}/{path}")
```

(error says "only single s3 paths are supported")

**Comment by p-a-a-a-trick** [comment]
@cboettig thanks for idea, code, and dataset. See #14 - `ibis.read_parquet` now works due to some recent changes (should be in 5.1 if not already in 5.0).

Unfortunately I couldn't get register_table to work, but IMO not a crushing defeat for the concept/example.

**Comment by jcrist** [comment]
Fixed by #14.
